core/src/main/resources/oracle/weblogic/deploy/aliases/category_modules/VirtualHost.json

```diff
@@ -7,6 +7,8 @@
     "folders": {
         "WebServerLog": {
             "wlst_type": "WebServerLog",
+            "child_folders_type": "single_unpredictable",
+            "default_name_value": "%VIRTUALHOST%",
```

Comment:
for s/u type, the wlst_paths entry needs a separate token for the last element, similar to others:

"WP001": "/VirtualHost${:s}/%VIRTUALHOST%/WebServerLog/%WEBSERVERLOG%"